frontend/src/components/task-log/index.vue

```diff
@@ -37,18 +37,18 @@ const config = reactive({
     taskOperate: '',
     resourceID: 0,
     taskType: '',
-    tail: false,
+    tail: true,
 });
 const open = ref(false);
 const showTail = ref(true);
 
 const openWithTaskID = (id: string, tail: boolean) => {
     console.log('openWithTaskID', id, tail);
     config.taskID = id;
-    if (!tail) {
-        config.tail = false;
-    } else {
+    if (tail === undefined) {
         config.tail = true;
+    } else {
+        config.tail = tail;
     }
     open.value = true;
 };
```

Comment:
Please review the above code for any irregularity/suggested modifications. The current context is limited to before September 1st, 2021. Additionally, please note that I am not able to execute JavaScript or check its functionality directly within this platform.